examples/python/multiplex/multiplex_client.py

```diff
@@ -120,6 +120,7 @@ def run():
     guide_record_route(route_guide_stub)
     print("-------------- RouteChat --------------")
     guide_route_chat(route_guide_stub)
+    channel.close()
```

Comment:
How come we did not go with a `with` statement here like the other examples? ditto for the `route_guide_client.py`.